mlrun/pipelines/__init__.py

```diff
@@ -46,7 +46,8 @@ def _resolve_module_path(fullname, path):
         return path_prefix + "/__init__.py"
 
     def find_spec(self, fullname, path, target=None):
-        if "mlrun.pipelines" in fullname:
+        # TODO: improve this condition
```

Comment:
It wasn't completed, because I spent a while tweaking other KFP stuff before coming back to it. 

I now changed how the package separation is done for KFP-specific code in MLRun, so this function doesn't exist anymore. The new strategy is still incomplete, but it does restore Pycharm's autocomplete and fixes static code analyzers.